barman/backup.py

```diff
@@ -371,6 +371,27 @@ def get_last_backup_id(self, status_filter=DEFAULT_STATUS_FILTER):
         ids = sorted(available_backups.keys())
         return ids[-1]
 
+    def get_last_full_backup_id(self, status_filter=DEFAULT_STATUS_FILTER):
+        """
+        Get the id of the latest/last FULL backup in the catalog (if exists)
+
+        :param status_filter: The status of the backup to return,
+            default to :attr:`DEFAULT_STATUS_FILTER`.
+        :return str|None: ID of the backup
+        """
+        available_full_backups = dict(
+            filter(
+                lambda backup: backup[1].is_full_and_eligible_for_incremental(),
+                self.get_available_backups(status_filter).items(),
```

Comment:
```suggestion
                lambda backup: backup.is_full_and_eligible_for_incremental(),
                self.get_available_backups(status_filter).values(),
```

I think this will look cleaner than `items` + `[1]`.